docs/versioned_docs/version-0.72.0/quickstart.md

```diff
@@ -840,7 +840,7 @@ def insert_data(plan, data):
 Now clean and run, only this time with extra args to `kurtosis run`:
 
 ```bash
-kurtosis clean -a && kurtosis run --enclave quickstart . '[{"first_name":"Kevin", "last_name": "Bacon"}, {"first_name":"Steve", "last_name":"Buscemi"}]'
+kurtosis clean -a && kurtosis run --enclave quickstart . '{"data": [{"first_name":"Kevin", "last_name": "Bacon"}, {"first_name":"Steve", "last_name":"Buscemi"}]}'
```

Comment:
Any chance we can change `data` to something more on topic here? With `data`, it feels a bit like a hack, whereas if we had something like `{persons: [kiven, steve, etc.]}` mayyybe that'd be nicer? 
Deferring to you here